packages/react-native-hid/src/index.js

```diff
@@ -9,6 +9,34 @@ type DeviceObj = {
 };
 
 export default class HIDTransport extends Transport<DeviceObj> {
+  static observer: *;
+  static unsubscribed: boolean;
+
+  id: number;
+
+  constructor(id: number) {
+    super();
+    this.id = id;
+  }
+
+  static onDeviceConnect = (device: *) => {
+    if (!HIDTransport.unsubscribed) {
+      const deviceModel = identifyUSBProductId(device.productId);
+      HIDTransport.observer && HIDTransport.observer.next({
+        type: "add",
+        descriptor: device,
+        deviceModel
+      });
+    }
+  };
+
+  static onDeviceDisconnect = (_: *) => {
+    if (HIDTransport.observer && !HIDTransport.unsubscribed) {
+      HIDTransport.observer.next({ type: "reset" });
+      HIDTransport.listDevices();
+    }
+  };
```

Comment:
same for the 2 callbacks. can be functions outside the class.

Reply:
ok